packages/shared/src/logger/default-logger.ts

```diff
@@ -3,6 +3,15 @@
 import type { Logger } from './logger';
 
 export class DefaultLogger implements Logger {
+  
+  private readonly showInfo : boolean = false;
+  private readonly showDebug : boolean = false;
+
+  constructor(showInfo: boolean = false, showDebug: boolean = false){
```

Comment:
Perhaps a log-level argument could work well. It could default to "warn" by default.

Reply:
@beeme1mr This has been addressed with latest commit.